test/Dialect/FIRRTL/emit-basic.mlir

```diff
@@ -276,6 +276,24 @@ firrtl.circuit "Foo" {
     // CHECK-NEXT:  MyMem.b.clk <= someClock
     // CHECK-NEXT:  MyMem.c.clk <= someClock
 
+    %combmem = firrtl.combmem : !firrtl.cmemory<uint<3>, 256>
+    %port0_data, %port0_port = firrtl.memoryport Infer %combmem {name = "port0"} : (!firrtl.cmemory<uint<3>, 256>) -> (!firrtl.uint<3>, !firrtl.cmemoryport)
+    firrtl.when %ui1 {
+      firrtl.memoryport.access %port0_port[%someAddr], %someClock : !firrtl.cmemoryport, !firrtl.uint<8>, !firrtl.clock
```

Comment:
I would add a use of `%port0_data` and `%port1_data` somewhere in here, to make sure the emitter knows how to refer to the ports when they are used in other operations.